floflis

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # load definitions & settings
-source /usr/sbin/layers/dna/config
```

Comment:
the FHS says that it is bad to put folders in /bin /usr/bin /sbin and /usr/sbin

Reply:
I think /usr/bin is better than /usr/sbin